src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.Tagger.cs

```diff
@@ -30,6 +30,8 @@ private sealed partial class Tagger : IAccurateTagger<TTag>, IDisposable
 
             private readonly ITextBuffer _subjectBuffer;
 
+            private readonly CancellationTokenSource _cancellationTokenSource;
```

Comment:
so this is only called when tagger goes away right? shutdownTokenSource or disposedTokenSource?

Reply:
Correct